commands/local/generate/fixture.js

```diff
@@ -13,11 +13,15 @@ module.exports.builder = function builder(yargs) {
 
 module.exports.handler = function handler(options) {
   const fs = require('fs-extra');
+  const { getTransformType } = require('../../../src/transform-support');
 
   let { codemodName, fixtureName } = options;
   let codemodDir = `${process.cwd()}/transforms/${codemodName}`;
+  let codemodTransform = `${codemodDir}/index.js`;
```

Comment:
Do we need to add `index.js` here? I wonder if we can basically just let it fall through via the `require` that is done down in `getTransformType` (since `require(codemodDir)` and `require(codemodDir + '/index.js')` are effectively the same thing)? That would allow folks to use other extensions (for example `.ts`)...

Reply:
Done